tests/sqlx/lexer.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { constructSyntaxTree, parseSqlx, lexer } from "@dataform/sqlx/lexer";
+import { expect } from "chai";
+
+describe("@dataform/sqlx", () => {
+
+  describe("lexer", () => {
+    it("sqlx_single_backslash_stays_as_double", () => {
+      // These look like groups of 2 backslashes, but are actually 1. If someone
+      // in SQLX SQL writes "/" in the regex, it should be interpreted as "/".
```

Comment:
this comment (and below) refers to forward-slashes, i think you want backslashes.

Reply:
👍